src/index.html

```diff
@@ -6,12 +6,179 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="/action_page.php" method="post">
+      <fieldset>
+        <legend>Personal information
+        </legend>
+        <section>
+          <label required for="surname">Surname:
+          </label>
```

Comment:
```suggestion
          <label required for="surname">
            Surname:
          </label>
```
or like this - fix everywhere